radicle-httpd/src/api/v1/sessions.rs

```diff
@@ -82,13 +103,14 @@ async fn session_signin_handler(
             .checked_add(AUTHORIZED_SESSIONS_EXPIRATION)
             .unwrap();
         let session = Session {
+            status: AuthState::Authorized,
             public_key: request.pk,
-            issued_at: DateTime(OffsetDateTime::now_utc()),
+            issued_at: session.issued_at.to_owned(),
             expires_at: DateTime(expiration_time),
         };
-        sessions.insert(session_id.clone(), AuthState::Authorized(session));
+        sessions.insert(session_id.clone(), session);
 
-        return Ok::<_, Error>(());
+        return Ok::<_, Error>(Json(json!({ "success": "true" })));
```

Comment:
What's the idea here, isn't `200 OK` sufficient?

Reply:
e.g. if you use a cli cmd, and returns nothing, you don't know whether it suceeded or not, there's always `echo $` but actual textual output is nicer.

Reply:
Yeah, guess that's useful. Let's use a boolean then, so `{ "success": true }`.

Reply:
has been done in 6f62e5c

Reply:
the[ `true` part is string](https://github.com/radicle-dev/heartwood/commit/6f62e5cb93ff288160c9e56e00127a39f49689ab#diff-83176232347eff805dc12f6bbb8553f21f50e79f430bc2d7248c5a93c930d45aR112-R114), want it changed to boolean:
```
"success": "true" -> "success": true
```

Reply:
you could also extract it and re-use `api::json::success` in both places instead.